### PR TITLE
Fix import form alignment

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | See issue.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/10686
| How to test?  | Check issue.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10741)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/ImportPage/import.html.twig [hunk 1]
```diff
@@ -27,31 +27,33 @@
 {% trans_default_domain 'Admin.Advparameters.Feature' %}
 
 {% block content %}
-    {% if importForm is defined %}
-    <div class="container">
+  {% if importForm is defined %}
```
Comment (PierreRambaud) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/ImportPage/import.html.twig:30]: why two spaces here and four spaces above?

Comment (sarjon) [reply src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/ImportPage/import.html.twig:30]: i cant see 4 spaces above. :/ what do you mean?

Comment (PierreRambaud) [reply src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/ImportPage/import.html.twig:30]: Sorry speaking about `import_panel.html.twig` file line 34 / 35 6 spaces and line 167 / 168 there is 4 spaces

Comment (sarjon) [reply src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/ImportPage/import.html.twig:30]: done.

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig [hunk 2]
```diff
@@ -28,152 +28,149 @@
...
+      {{ ps.label_with_help(('Language of the file'|trans), ('The locale must be installed'|trans({}, 'Admin.Advparameters.Notification'))) }}
+      {{ form_errors(importForm.iso_lang) }}
+      {{ form_widget(importForm.iso_lang) }}
+    </div>
+
+    <div class="form-group">
+      <label class="form-control-label">{{ 'Field separator'|trans }}</label>
+      {{ form_errors(importForm.separator) }}
+      {{ form_widget(importForm.separator) }}
+      <small class="form-text">{{ 'e.g. '|trans({}, 'Admin.Advparameters.Help') }} 1; Blouse; 129.90; 5</small>
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig:121]: There is a blank space at the end of the string.

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig [hunk 3]
```diff
@@ -28,152 +28,149 @@
...
+    </div>
+
+    <div class="form-group">
+      <label class="form-control-label">{{ 'Multiple value separator'|trans }}</label>
+      {{ form_errors(importForm.multiple_value_separator) }}
+      {{ form_widget(importForm.multiple_value_separator) }}
+      <small class="form-text">{{ 'e.g. '|trans({}, 'Admin.Advparameters.Help') }} Blouse; red.jpg, blue.jpg, green.jpg; 129.90</small>
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig:128]: Same here.

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig (hunk 2 above)
Comment (sarjon) [reply src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig:121]: translation key contains this space. :)

**Review by matks — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @sarjon

**Pull request merged**